geometry_msgs/Point[] points
geometry_msgs/PoseStamped center_pose
int32 angle
float32 short_radius
float32 long_radius
float32 length_to_center
float32 score
int32 index